src/index.html

```diff
@@ -10,13 +10,53 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    >
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header>
+    <a
+      href="#"
+      class="logo"
+    >
+      <img
+        src="images/logo.png"
+        alt="logo"
+      />
+    </a>
+    <nav>
+      <ul>
+        <li>
+          <a
+            href="#"
+            class="is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li><a href="#">Samsung</a></li>
+        <li><a href="#">Smartphones</a></li>
+        <li>
+          <a
+            href="#"
+            data-qa="hover"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+        <li><a href="#">Gadgets</a></li>
+        <li><a href="#">Tablets</a></li>
+        <li><a href="#">Photo</a></li>
+        <li><a href="#">Video</a></li>
+      </ul>
```

Comment:
Class names represent the meaning of the content. Instead of generic classes like 'is-active', use more descriptive class names that represent the content or the role of the element, such as 'nav__link--active'.